tests/unittests/CoreGraphics/CGBitmapContextTests.mm

```diff
@@ -65,4 +65,105 @@
 
     CGColorSpaceRelease(grayColorSpace);
     CGContextRelease(context);
-}
\ No newline at end of file
+}
+
+void _DrawText(CGContextRef context, NSString* text, const CGRect bounds) {
+    // Aligns origin for our frame
+    CGContextTranslateCTM(context, 0.0f, bounds.size.height);
+
+    // Flips y-axis for our frame
+    CGContextScaleCTM(context, 1.0f, -1.0f);
+
+    // Creates path with current rectangle
+    CGMutablePathRef path = CGPathCreateMutable();
+    CGPathAddRect(path, NULL, bounds);
```

Comment:
or put into a `woc::unique_cf`! :)

---
In reply to: [84944101](https://github.com/Microsoft/WinObjC/pull/1220#discussion_r84944101) [](ancestors = 84944101)